### PR TITLE
Add support for --device option in docker module

### PR DESCRIPTION
**Pull request by ghost**
implements #1338

### CODE REVIEW

**Comment by jquadrin** [comment]
@bcoca this is a very simple feature

**Comment by paulfitz** [comment]
+1 this would be great to have

**Comment by bcoca** [comment]
cc for review @smashwilson @cove @softzilla @joshuaconner

**Comment by jluan** [comment]
:+1:

**Comment by chuckcho** [comment]
:+1:

**Comment by gregdek** [comment]
@jquadrin Thanks for submitting this pull request. Unfortunately, it appears that the underlying code has changed since your submission, and this PR now needs to be rebased. 

If you believe that this PR still applies, please rebase and we will put it into our community review process as soon as possible. 

Once you have rebased this PR, please add a comment with the text "ready_for_review" and we will continue with the review process.  Thanks!

**Comment by smashwilson** [comment]
@jquadrin Apologies for the delay, it's been a (very) long time since I've found time to go through these. Your code looks solid, so ping me if and when you get around to doing a rebase, and I'll thumbs-up this one to move it along.

File: cloud/docker/docker.py [hunk 1]
```diff
@@ -92,6 +92,10 @@
       - 'alias. Use docker CLI-style syntax: C(redis:myredis).'
     default: null
     version_added: "1.5"
+  devices:
+    description:
+      - List of host devices to expose to container
+    default: null
```
Comment (smashwilson) [inline cloud/docker/docker.py:98]: While you're rebasing, could you add `required: false` to the documentation YAML here?

**Comment by RaymondKlass** [comment]
looks good to me

**Comment by smashwilson** [comment]
Excellent, thanks. :shipit:

**Comment by ghost** [comment]
@smashwilson rebased, ready to be merged :+1: thanks!

**Comment by ghost** [comment]
ready_for_review

**Comment by gregdek** [comment]
@smashwilson gives the shipit frog, so we're good to go. :)

Thanks @jquadrin for the PR. Marking for potential inclusion.

**Comment by birish2** [comment]
Will this be merged anytime soon?

**Comment by abadger** [comment]
Apologies.  We've been busy for what feels like months trying to get the ansible-2.0 release out the door.

I've now merged this into the devel branch https://github.com/ansible/ansible-modules-core/commit/8fe5d6f7ef7f17f1ec54a44570daf28ebd85f208  so it will be released with ansible-2.1.

# Merged

Hi!

This has been merged in, and will also be included in the next major release.

If you or anyone else has any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!

**Pull request closed without merging**